.github/workflows/verify-generated-code-changes.yml

```diff
@@ -0,0 +1,43 @@
+name: Verify generated code changes
+
+on:
+  workflow_dispatch:
+  pull_request:
```

Comment:
Should we do on PR or on merge ? Merge wouldn't run as much which is good, but then user might not see it fail.